### ISSUE TITLE
hope support postfix

### ISSUE THREAD

**Issue by liuwangchen** [golang/vscode-go#144]
hope
![image](https://user-images.githubusercontent.com/25526033/83398308-b1b4f280-a431-11ea-9119-bf8e9117e97e.png)
↓
![image](https://user-images.githubusercontent.com/25526033/83398337-bf6a7800-a431-11ea-9747-81d278801b91.png)

**Comment by hyangah** [comment]
@liuwangchen Do you mean that you want a certain word at the end of a function call triggers autocompletion of LHS? In your example, `.var` is the certain word. Did I guess correctly?

**Comment by liuwangchen** [comment]
Yes, I used goland to find this nice feature, and I hope this plugin will support it

**Comment by stamblerre** [comment]
Thanks for the feature request! We likely will not be making improvements to the completions provided by this extension directly - rather, we will make changes in [`gopls`](https://golang.org/s/gopls). I will transfer this issue to https://github.com/golang/go as a `gopls` feature request.
